styles/SASS/_pokemon-choose-view.scss

```diff
@@ -1 +1,64 @@
-// Styles for pokemon choosing screen. Common elements should be added to _config.scss in mixins.
\ No newline at end of file
+// Styles for pokemon choosing screen. Common elements should be added to _config.scss in mixins.
+
+#choose-page {
+    width: 100%;
+    height: 100%;
+    display: flex;
+    flex-flow: column nowrap;
+    align-items: center;
+    justify-content: center;
+    align-content: center;
+    padding: 5vh 2.5vh;
+
+    > * {
+        flex: 1 1 20%;
+        text-align: center;
+        font-weight: 900;
+    }
+
+    p {
+        font-size: 2.2vh;
```

Comment:
dont use vh or vw for font-size - this way is hard to control.
We set **root (html) font-size to 10px** so u can simply calculate it. 
Default for body is atm 2.4 em  = 24px. 
If u want font-size be equal to 44px for h1 set it 4.4rem or for 28 px it will be 2.8rem